devlink_linux.go

```diff
@@ -285,6 +451,38 @@ func (h *Handle) createCmdReq(Socket string, cmd uint8, bus string, device strin
 	return f, req, nil
 }
 
+// createCmdReqMod is like createCmdReq but uses modified netlink constructs
```

Comment:
can you add @TODO that this will be deprecate/remove once we fix the netlink issue in the kernel.

Reply:
done.